javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -30,7 +30,10 @@ Let's use what we've learned and take a chance to continue practicing DOM manipu
 1. Create a bare-bones homepage for a restaurant. Include an image, headline, and some copy about how wonderful the restaurant is. It’s okay to hard-code these into the HTML for now just to see how they look on the page.
 
 
-1. Now remove those elements from the HTML (so leave only the `<html>`, `<body>`, and `<div id="content">` tags) and instead create them by using JavaScript only, e.g. by appending each new element to `div#content` once the page is first loaded. Since we're all set up to write our code in multiple files, let's write this initial page-load function inside of its own module and then import and call it inside of `index.js`.
+1. Now remove those elements from the HTML (so leave only the `<html>` and `<body>`tags) and instead create them by using JavaScript only, e.g. by appending each new element to the body once the page is first loaded. 
+    1. If you're using the `HtmlWebpack`, you will need to edit your `src/index.js` file to insert the `div#content` into the body; the webpack creates a blank HTML page whic will remove your `div#content`
```

Comment:
```suggestion
    1. If you're using the `HtmlWebpackPlugin`, you will need to edit your `src/index.js` file to insert the `div#content` into the body; Webpack creates a new HTML page which will remove your `dist/index.html` file.
```

The previously mentioned edits from @Baguirre03, with a couple new ones from me to improve clarity as the page won't (at least I don't think) be blank, and it will replace the whole file rather than just that specific section.

We might need to change this a lot more depending on what changes we make to the top level bullet here though.